### ISSUE TITLE
I was released to the app store and was rejected.This library only uses swift in my project.

### ISSUE THREAD

**Issue by skyliwq** [edeckers/react-native-blob-courier#144]
I was released to the app store and was rejected.This library only uses swift in my project.

ITMS-90433: Invalid Swift Support - The file libswiftCore.dylib doesn’t have the correct code signature. Make sure you’re using the correct signature, rebuild your app using the current public (GM) version of Xcode, and resubmit it. Don’t just modify the code signature of libswiftCore.dylib.

Best regards,

The App Store Team

**Comment by edeckers** [comment]
Hi @skyliwq, thank you for reporting this issue! I'll check it out as soon as possible, hopefully this week.

Meanwhile, can you check if [this](https://stackoverflow.com/a/56591141) helps? In particular:
> add swift embed library always to YES, otherwise you can go to XCode build setting and set a flag to YES?

**Comment by edeckers** [comment]
Today I tried to reproduce your problem by publishing the example app included in this repository to Testflight from the `develop` branch and it worked just fine. I tested it with `Always Embed Swift Standard Libraries` / `ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES` set to both `YES` and `NO`, which didn't  make any difference.

What version of React Native are you using? And what is your iOS target version?
